tech/database/cassandra/cassandra_language_connectors.md

```diff
@@ -0,0 +1,4 @@
+## Language connectors/adapters/drivers (packages)
+
```

Comment:
Please add some entry on how to install them / what they are instead of link to package sources.